src/hub.html

```diff
@@ -416,7 +416,7 @@
                     </a-entity>
                     <a-entity class="label-background" mixin="rounded-button" is-remote-hover-target tags="singleActionButton: true;" position="0 0.15 0.08" scale="0.75 0.75 0.75">
                     </a-entity>
-                    <a-entity class="ui interactable-ui camera-snap-menu" mixin="button-container" visible-if-permitted="spawn_camera">
```

Comment:
The `button-container` mixin was removed at some point in the past, so this was doing nothing.